src/controllers/FrameController.ts

```diff
@@ -410,6 +410,17 @@ export class FrameController {
         return res.assignImageFromUrl(imageFrameId, url).then((result) => getEditorResponseData<null>(result));
     };
 
+    /**
+     * This method will assign an image from a variable to the correct ImageFrame
+     * @param imageFrameId The ID of the imageFrame where an image needs to be assigned to
+     * @param variableId The ID of the variable which contains the image
+     * @returns
+     */
+    setImageFromUrlVariable = async (imageFrameId: Id, variableId: string) => {
+        const res = await this.#editorAPI;
+        return res.assignImageVariable(imageFrameId, variableId).then((result) => getEditorResponseData<null>(result));
+    };
+
```

Comment:
Should be first placed in the experimentController `src/controllers/ExperimentController.ts`
Also would call it insertImageVariableToFrame istd. of setImageFromUrlVariable